_docs/howto/configure/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Configure Forseti Security Features
+order: 101
```

Comment:
In previewing this all the articles are listing under deploy in the left sidebar. To get this to work we need a new leading digit for articles in `/configure/...`.

This would become `order: 201`.